README.md

```diff
@@ -49,3 +49,18 @@ To only translate the CQL and not do the rest of the build/test steps, spin up a
 docker run -d -p 8080:8080 cqframework/cql-translation-service
 yarn translate
 ```
+
+### Utility Function Assurance Testing
+
+The `assurance` folder contains unit tests and relevant fixtures used for testing the functionality of the helper functions used in the testing harness. 
```

Comment:
```suggestion
The `assurance` folder contains tests and fixtures for testing the functionality of the helper functions used in the testing harness. 
```

A few edits for clarity. Mainly, I want to avoid calling these unit tests until we properly rename the other suite of tests to be something other than `test:unit`

Reply:
I think this would be a good thing to include as well, holding off to see if Matt thinks so too though!